df.language.xml

```diff
@@ -394,9 +394,9 @@
         <enum-item name='Bridge'/>
         <enum-item name='Tunnel'/>
         <enum-item name='PretentiousEntityPosition'/>
-        <enum-item name='Unk37'/>
+        <enum-item name='Monument'/>
         <enum-item name='Tomb'/>
-        <enum-item name='MigratingGroup'/>
+        <enum-item name='OutcastGroup'/>
```

Comment:
Out of curiosity, what's the reason for the name change? I'm not quite seeing the distinction (and strangemood did already use the old name)

Reply:
The NomadicGroup enum value is used by Strangemood as well, so the code had to be modified anyway, or I would probably have refrained from changing this enum value name.

The reason was that all of them (at least in the saves I looked at) were of the entity type Outcast, while the newly identified one was of the type Nomadic when it hadn't migrated to found a site (in which case they were SiteGovernment [and I haven't investigated if they change type when doing so or if they are just created as a Site Government directly: those connections are often quite hazy in Legends Mode info]). Thus, I found it clearer to identify the enum with a name that was much closer to the type of entity that used those names, rather than one that would be confusingly similar in meaning to the newly named one.

Reply:
Ah, so you're matching `historical_entity_type`. Sounds good, and thanks for the explanation!

I am usually a bit hesitant to rename things that already have non-temporary names, especially when they're already used. If something is already used by a plugin/script, it tends to be more likely used in others as well, and scripts aren't statically-checked. I didn't see any uses of `language_name_type` in scripts, though, so I'm okay with this.

Reply:
Yes, it's a somewhat obscure enum type that would be used only where you're trying to name things according to the same rules as DF does, using the same set of words, but I admit to not think far ahead to check scripts, so that's useful for the future (assuming the memory is with me at the time...).